README.md

```diff
@@ -20,15 +20,200 @@ tasks specified in the Gruntfile.
 
 #### Prerequisites
 
-- Java
-- NodeJS (12 or higher)
-- Grunt
-- Android SDK
-
-For a more complete description of how to use this repository, please see our
-user documentation at:
-
-[https://docs.odk-x.org/app-designer-intro/](https://docs.odk-x.org/app-designer-intro/)
+- Java -> Java is required by the Android SDK (installation is described below).
+- Chrome -> [Google's Chrome browser](https://www.google.com/intl/en/chrome/browser/desktop/index.html). 
+- NodeJS -> a framework for easily building fast, scalable applications (installation is described below).
+- Grunt -> a task-based scripting environment (installation is described below).
+- Android SDK -> the software development kit for Android devices (installation is described below).
+
+##### Java
+Make sure Java 8 or higher is installed on the computer you plan to use.  If it is not, [download and install it](https://java.com/en/download/). If you are using MacOSX, it may require special care and attention.See [MacOSX Java install](https://docs.oracle.com/javase/7/docs/webnotes/install/mac/mac-jdk.html) and [MacOSX Java FAQ](https://docs.oracle.com/javase/7/docs/webnotes/install/mac/mac-install-faq.html).
```

Comment:
The links for macOS point to Java 7 docs, since we require Java 8 or above please update to a more up to date reference.